src/views/home/sections/MentorsVolunteers/MentorsVolunteers.tsx

```diff
@@ -1,75 +1,36 @@
 	return (
 		<Row>
-			<Col>
-				<div className={styles.statueContainer}>
-					<Image src={mentorStatue} alt="HackUCI Mentor Statue" />
-					<h3 className={styles.statueTitle}>Apply to be a Mentor</h3>
-					<div className={styles.statueCube}>
-						<div className={`${styles.cubeSide} ${styles.cubeTop}`} />
-						<div className={`${styles.cubeSide} ${styles.cubeLeft}`} />
-						<div
-							className={`${styles.cubeSide} ${styles.cubeFront} ${styles.statueDescription}`}
-						>
-							<p>
-								Have a knack for innovation? Interested in inspiring the next
-								generation of developers? Our mentors are vital to making
-								HackUCI 2023 successful and enjoyable for our hackers. Apply to
-								be a mentor today!
-							</p>
-							<Button
-								variant="museum"
-								href="/mentor"
-								role=""
-								target="_blank"
-								rel="noreferrer noopener"
-							>
-								Mentor Application
-							</Button>
-						</div>
-					</div>
-				</div>
-			</Col>
-			<Col>
-				<div className={styles.statueContainer}>
-					<Image src={volunteerStatue} alt="HackUCI Volunteer Statue" />
-					<h3 className={styles.statueTitle}>Become a Volunteer</h3>
-					<div className={styles.statueCube}>
-						<div className={`${styles.cubeSide} ${styles.cubeTop}`} />
-						<div className={`${styles.cubeSide} ${styles.cubeLeft}`} />
-						<div
-							className={`${styles.cubeSide} ${styles.cubeFront} ${styles.statueDescription}`}
-						>
-							<p>
-								Want a peek behind the scenes? Join our team of event volunteers
-								that make HackUCI run as smoothly as possible! Many roles are
-								available and shifts are flexible. Plus, get access to all the
-								free food and all our hackathon swag. We hope to see you on the
-								team!
-							</p>
-							<Button
-								variant="museum"
-								href="/volunteer"
-								role=""
-								target="_blank"
-								rel="noreferrer noopener"
-							>
-								Volunteer Form
-							</Button>
-						</div>
-					</div>
-				</div>
-			</Col>
+			<StatuePedestal
+				statueTitle="Apply to be a Mentor"
+				statueImage={mentorStatue}
+				statueAlt="HackUCI Mentor Statue"
+				href="/mentor"
+				buttonName="Mentor Application"
+			>
+				Have a knack for innovation? Interested in inspiring the next generation
+				of developers? Our mentors are vital to making HackUCI 2023 successful
+				and enjoyable for our hackers. Apply to be a mentor today!
+			</StatuePedestal>
+			<StatuePedestal
+				statueTitle="Become a Volunteer"
+				statueImage={volunteerStatue}
+				statueAlt="HackUCI Volunteer Statue"
+				href="/volunteer"
+				buttonName="Volunteer Form"
+			>
+				Want a peek behind the scenes? Join our team of event volunteers that
+				make HackUCI run as smoothly as possible! Many roles are available and
+				shifts are flexible. Plus, get access to all the free food and all our
+				hackathon swag. We hope to see you on the team!
+			</StatuePedestal>
```

Comment:
Suggestion: I think the abstraction is too coupled right now: the button could be specified as part of the children, not constrained to the pedestal.